pkg/24.0.0/config.sh

```diff
@@ -5,6 +5,6 @@ DOCKER_GIT_COMMIT="c2de0811708b6d9015ed1a2c80f02c9b70c8ce7b"
 TEST_OS_IMAGE_NAME=(redhat/ubi9 oraclelinux centos ubuntu rockylinux)
 TEST_OS_IMAGE_TAG[0]="9.0.0"
 TEST_OS_IMAGE_TAG[1]="7 8"
-TEST_OS_IMAGE_TAG[2]="centos7 centos8"
+TEST_OS_IMAGE_TAG[2]="centos7"
```

Comment:
Can we have CentOS 8 Stream added to the automated testing?

Reply:
Added Centos Stream8. The images are present only on quay.io